build/pack.c

```diff
@@ -14,6 +14,8 @@
 #include <rpm/rpmfileutil.h>
 #include <rpm/rpmlog.h>
 
+#include <prtpool.h>			/* NSPR thread pools */
```

Comment:
@kanavin NSS is not hard dependency, it has support for openssl and other crypto backend and I would say that NSS is hardest to bootstrap so we swithced to OpenSSL in Fedora not so long time ago... So assuming NSS to be present is very bad idea.